tests/automation/disappearing_message_checks.spec.ts

```diff
@@ -237,14 +237,16 @@ test_Alice_1W_Bob_1W(
     // To stop the layout shift
     await sleepFor(500);
     await clickOnTestIdWithText(aliceWindow1, 'conversation-options-avatar');
-    await clickOnTestIdWithText(aliceWindow1, 'add-user-button');
+    await clickOnTestIdWithText(aliceWindow1, 'invite-contacts-menu-option');
     await waitForTestIdWithText(
       aliceWindow1,
       'modal-heading',
       englishStrippedStr('membersInvite').toString(),
     );
     await clickOnTestIdWithText(aliceWindow1, 'contact', bob.userName);
     await clickOnTestIdWithText(aliceWindow1, 'session-confirm-ok-button');
+    // For lack of a unique ID we use native Playwright methods 
+    await aliceWindow1.getByRole('dialog').filter({ hasText: 'Invite Contacts' }).getByTestId('modal-close-button').click();
```

Comment:
what is this last char? 
![image](https://github.com/user-attachments/assets/65457ff7-3abf-4b52-a9b6-8cd9315038cf)